client-js/app/elements/labrad-registry.ts

```diff
@@ -0,0 +1,195 @@
+import {RegistryApi, RegistryListing} from '../scripts/registry';
```

Comment:
maybe; I wanted to keep them close to the html files because they're tightly coordinated with each other. We can certainly discuss reorganizing things.